File: 백준 Python/Step5 실습1/pro5_1.py
import sys
score = []
for i in range (5) :
    a = int(sys.stdin.readline())
    if a <= 40 :
        a = 40
    score.append(a)
print((score[0]+score[1]+score[2]+score[3]+score[4])//5)
    

File: 백준 Java/Step5 실습1/src/Pro5_1.java
import java.io.*;

public class Pro5_1 {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int[] score = new int[5];
		for (int i=0; i<5; i++) {
			int a = Integer.parseInt(br.readLine());
			if(a<=40) {
				a = 40;
			}
			score[i] = a;
		}
		System.out.println((score[0]+score[1]+score[2]+score[3]+score[4])/5);
	}

}
